sources/packages/api/src/cra-integration/cra-integration.service.spec.ts

```diff
@@ -44,6 +45,7 @@ describe("CRAIntegrationService", () => {
         givenName: faker.name.firstName(),
         birthDate: faker.date.past(18),
         taxYear: faker.random.number({ min: 2000, max: 2030 }),
+        userId: createFakeUser().id,
```

Comment:
_createFakeUser_ is not actually saving the object to the DB so the id is not supposed to be present here. Is this code working?

Reply:
removed it, as changed the logic